pkg/utils/ocm.go

```diff
@@ -432,28 +432,56 @@ func GetHiveCluster(clusterId string) (*cmv1.Cluster, error) {
 	return resp.Items().Get(0), nil
 }
 
-// GetManagementCluster returns the OCM Cluster object for a provided clusterId
-func GetManagementCluster(clusterId string) (*cmv1.Cluster, error) {
+// GetMgmtServiceCluster returns the Mgmt Cluster and Service Cluster object for a provided HCP clusterId
+func GetMgmtServiceCluster(clusterId string) (*cmv1.Cluster, *cmv1.Cluster, error) {
```

Comment:
nit pick: Wondering if `GetManagementAndServiceCluster` would be a better name. It is a bit long but is clear enough

Reply:
or is it good to add another func to get service cluster by clusterID, 
otherwise, we may call 2 API calls when it's only needed MC cluster